jack-matchmaker.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=auto-connect JACK ports matching given patterns
+
+[Service]
+EnvironmentFile=/etc/conf.d/jack-matchmaker
+ExecStart=/usr/bin/jack-matchmaker -p "$PATTERN_FILE" -e $EXACT_MATCHING -N "$CLIENT_NAME" -m $MAX_ATTEMPTS -v $VERBOSITY "$PATTERNS"
```

Comment:
The `-e` option does not take arguments, so `-e $EXACT_MATCHING` will not work as intended.

The only way to set this I see is to have something like `EXTRA_OPTIONS="-e"` in the EnvironmentFile and then add `$EXTRA_OPTIONS` to the command line (not quoted!).

Reply:
`$PATTERNS` must not be quoted, so that the patterns are passed as separate command line arguments.

See section "Command Lines" [here](https://www.freedesktop.org/software/systemd/man/systemd.service.html).

Reply:
The `-e` option is fixed with parameter expansion: If EXACT_MATCHING is set to nothing it will not be presented as an argument.